.ci/unit-tests/inspec.sh

```diff
@@ -3,9 +3,66 @@
 set -e
 set -x
 
-pushd "magic-modules/build/inspec/test/unit"
+# Service account credentials for GCP to pull VCR cassettes
+export GOOGLE_CLOUD_KEYFILE_JSON="/tmp/google-account.json"
+
+# CI sets the contents of our json account secret in our environment; dump it
+# to disk for use in tests.
+echo "${TERRAFORM_KEY}" > /tmp/google-account.json
+
+export CLOUD_SDK_REPO="cloud-sdk-stretch"
+echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+apt-get update && apt-get install google-cloud-sdk -y
+
+gcloud auth activate-service-account terraform@graphite-test-sam-chef.iam.gserviceaccount.com --key-file=$GOOGLE_CLOUD_KEYFILE_JSON
+
+# Download train plugin (it's not published yet)
+gsutil cp -r gs://magic-modules-inspec-bucket/train-gcp2 .
+gem install inspec
+inspec plugin install train-gcp2/lib/train-gcp2.rb
+
+pushd "magic-modules/build/inspec/test/integration"
+
+# Generate a rsa private key to use in mocks
+# Due to using gauth library InSpec + train expect to load a service account file from an env variable
+# This service account file must contain a real RSA key, but this key is never used in unit tests.
+ssh-keygen -f id_rsa -t rsa -N ''
```

Comment:
In retrospect it'd be safer (or at least, more explicitly "this new key is not used for anything other than to have a valid-but-unused RSA key") to do something like:

```
rsatmp=$(mktemp /tmp/rsatmp.XXXXXX)
openssl genrsa -out "${rsatmp}"
...
cat ${rsatmp}
...
rm ${rsatmp}
```